pkg/cloudprovider/provider/gce/provider.go

```diff
@@ -47,6 +49,9 @@ const (
 	errOperatingSystem       = "Invalid or not supported operating system specified %q: %v"
 	errConnect               = "Failed to connect: %v"
 	errInvalidServiceAccount = "Service account is missing"
+	errIPv6UnsupportedZone   = "IPv6 is not supported in zone: %s"
+	errUnknownNetworkFamily  = "unknown network family only IPv4,IPv6,IPv4+IPv6 are valid values, got: %q"
```

Comment:
Looking at the pattern for those error messages: 
```suggestion
	errUnknownNetworkFamily  = "Unknown network family only IPv4,IPv6,IPv4+IPv6 are valid values, got: %q"
```